src/import-data/protected-data-utils.ts

```diff
@@ -25,6 +25,18 @@ export const isProtectedSource = (url: string, filetype?: string): boolean => {
   }
 };
 
+// This method identifies whether an import source is an AnVIL Explorer import
+export const isAnvilImport = (url: string): boolean => {
+  if (!url) {
+    return false;
+  }
```

Comment:
I don't think this check is necessary. Validation added in #4300 and TypeScript together guarantee that the input `url` is defined.

```suggestion
```

Reply:
If you look how I have to call it in ImportData, I dont think `url` is a guaranteed parameter of `importRequest` so I'll have to define an empty url (instead of an empty string). Should I check for empty or just have the function run and it'll determine that url does not contain <path>

Reply:
Oh, I missed that.

Alternatively, the call could be something like:
```ts
isAnvilData: 'url' in importRequest ? isAnvilImport(importRequest.url) : undefined
```

Reply:
ah yup that makes way more sense. ok function param types updated :)